index.js

```diff
@@ -692,35 +660,35 @@ class KeyringController extends EventEmitter {
    * @param {string} address - An account address.
```

Comment:
Nit: There is a `.then` a few lines up from here in `getAccounts` that we can remove as well